client/my-sites/sidebar/index.jsx

```diff
@@ -808,14 +809,41 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
+	renderSwitchSites() {
+		const { translate } = this.props;
+
+		if ( this.props.currentUser.site_count < 2 ) {
+			return null;
+		}
+
+		return (
+			<span className="sidebar__switch-sites">
+				<Button borderless onClick={ this.switchSites }>
+					<Gridicon icon="chevron-left" size={ 18 } />
+					<span className="sidebar__switch-sites-label">{ translate( 'Switch site' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **69** times:
`translate( 'Switch Site' )` *ES Score: 10*